### PR TITLE
test(PD): Add test suite for custom tip racks

### LINKED ISSUE #5633: tests: Add Cypress tests for Custom Tip Racks
# Overview
Add automated tests for custom tip racks to smooth the deploy process.

Write the following tests:

- [x] Upload invalid file type to custom tip rack file upload button
- [x] Upload invalid .json file
- [x] Upload valid labware that is not a tip rack
- [x] Upload valid tip rack
- [x] Upload valid tip rack again
- [x] Select the tip rack for either pipette
- [x] Duplicate tip rack
- [x] Add another of the same tip rack
- [x] Remove added tip rack from the deck

### PR DESCRIPTION
**Pull request by dozercodes**
## overview

Closes #5633. Adds a test suite cover the basic flow of adding a custom tip rack to a protocol. Tests covered: 

- Upload invalid file type to custom tip rack file upload button
- Upload invalid .json file
- Upload valid labware that is not a tip rack
- Upload valid tip rack
- Upload valid tip rack again
- Select the tip rack for both pipettes
- Duplicate tip rack
- Add another tip rack with add labware modal
- Remove added tip rack from the deck

## changelog

- Added E2E tests for custom tip racks
## review requests

Test structure and any other thoughts you have.
## risk assessment

None, just tests

### CODE REVIEW

**Review by Kadee80 — APPROVED** [review]
🎄

File: protocol-designer/cypress/integration/customTipRacks.spec.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import 'cypress-file-upload'
+
+// Common Variables and Selectors
+const protocolTitle = 'Custom Tip Rack Test Protocol'
+const pipette = 'P300'
+const customTipRackTitle = 'Custom 200µL Tiprack'
+const customTipRackTitleWithotMicro = 'Custom 200uL Tiprack'
+// Deck Map Slots
+const slotOne = 'foreignObject[x="0"][y="0"]'
```
Comment (IanLondon) [inline protocol-designer/cypress/integration/customTipRacks.spec.js:9]: we should make data attrs for these?

**Review by IanLondon — APPROVED** [review]

**Review by shlokamin — APPROVED** [review]
Looks solid!

File: protocol-designer/cypress/integration/customTipRacks.spec.js (hunk 1 above)
Comment (dozercodes) [reply protocol-designer/cypress/integration/customTipRacks.spec.js:9]: Agreed! They're on my list to add that I should share with the team

**Pull request merged**